pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -246,6 +246,10 @@ public void validatePoliciesReadOnlyAccess() {
     protected List<String> getListOfNamespaces(String property) throws Exception {
         List<String> namespaces = Lists.newArrayList();
 
+        if (!globalZkCache().exists(path(POLICIES, property))) {
+            throw new KeeperException.NoNodeException();
+        }
```

Comment:
Fix the issue introduced by https://github.com/apache/pulsar/pull/10594, and 2.7.3 also have this problem.